services/userService.js

```diff
@@ -35,7 +35,7 @@ const signIn = async (email, password) => {
         throw error
     }
 
-    return [userInfo[0].id, jwt.sign({ email: userInfo[0].email }, process.env.SECRET_KEY)]
+    return [userInfo[0].id,userInfo[0].username, jwt.sign({ email: userInfo[0].email }, process.env.SECRET_KEY)]
```

Comment:
```suggestion
    return {
      id: userInfo[0].id,
      username: userInfo[0].username,
      token: jwt.sign({ email: userInfo[0].email }, process.env.SECRET_KEY)
    }
```
이름을 바꿔야하는 경우가 아니라면 객체를 리턴하여 줍니다.
이름을 바꿔야하는 대표적인 경우는 프론트에서 사용하는 useState입니다.
```
function useState(initialValue) {
  let state, setState;
  ...useState 로직
  return [state, setState];
}

const [value, setValue] = useState();
```
이런식으로 구조 분해할당을 하시면 이름을 변경할 수 있기 때문에 array 형태로 반환하는 거고
이름을 바꿀 필요가 없을 경우에는 객체를 이용하시면 되겠습니다.
추가로 객체 구조 분해 할당도 이름을 변경할 수 있습니다만 array 형태보다는 깔끔하지 않습니다.